addons/udes_stock/models/stock_picking.py

```diff
@@ -271,23 +309,21 @@ def _backorder_move_lines(self):
         The reason for this method is to maintain tracability of move lines,
         and not delete and re-create them inside _action_done().
 
+        Assumes that _requires_backorder is called prior to the current method call.
```

Comment:
Yeah so _requires_backorder was dropped because of this. But with passing mls it is needed again.
Added a new method to unprivatise it.

Going to leave _requires_backorder as it is, because it only sometimes check if work is incomplete (mls=None) when it is set though, it checks if the ms fulfill the moves in the picking.(And some origin stuff on the moves but that's not ported so doesn't do much)

Reply:
Can leave this then?

Reply:
@LoKKeR96 @ws-mps @kdwyer We happy to resolve?

Reply:
In terms of the original comment, I think it's fine, @kdwyer should probably resolve it if he's happy as well.
I am not entirely happy about the name of "_requires_backorder()" but I'm not going to push for it so that's fine!